git-prompt.zsh

```diff
@@ -26,6 +26,11 @@ autoload -U colors && colors
 : "${ZSH_THEME_GIT_PROMPT_SEPARATOR="|"}"
 : "${ZSH_THEME_GIT_PROMPT_DETACHED="%{$fg_bold[cyan]%}:"}"
 : "${ZSH_THEME_GIT_PROMPT_BRANCH="%{$fg_bold[magenta]%}"}"
+: "${ZSH_THEME_GIT_PROMPT_UPSTREAM_TYPE="auto"}"
+: "${ZSH_THEME_GIT_PROMPT_UPSTREAM_COLOR="%{$fg[yellow]%}"}"
```

Comment:
Please don't introduce `*_COLOR` variables. Keep it consistent to how colors are handled in the existing prompt fragments, i.e. move coloring to to prefix and suffix variables.